lib/ui/text.dart

```diff
@@ -1569,6 +1569,185 @@ enum PlaceholderAlignment {
   middle,
 }
 
+/// Data class that holds the metrics for a single line in the [Paragraph].
```

Comment:
All comments I made on the previous file also apply here.

@jonahwilliams I thought we no longer repeat doc comments in the stub impl... Is the doc comment duplication still necessary?

Reply:
You don't need to repeat doc comments.

That said, I'm not enforcing this via a lint until there is an unfork, because the code in stub_ui is not the source of truth.

Reply:
@GaryQian Can you remove the comments from the stubs then?